src/main/io/ledstrip.h

```diff
@@ -28,7 +28,6 @@
 
 #include "pg/pg.h"
 
-#define LED_MAX_STRIP_LENGTH           32
```

Comment:
It's not possible to provide a define with the value into a cloud build tho

Reply:
I actually don't mind this (combo of both): 

```
#ifndef LED_MAX_STRIP_LENGTH
#ifdef USE_LED64
#define LED_MAX_STRIP_LENGTH           64
#else
#define LED_MAX_STRIP_LENGTH           32
#endif
#endif
```

That way you can set the value yourself (assuming everything else works) or use a simple option. I would add LED64 in options (but leaving LED). You can select LED64 or both - and it will use 64, but if you only select LED then it will only use 32.

Reply:
Lol, had almost same conclusion:

    #ifndef LED_MAX_STRIP_LENGTH
        #if defined(USE_64_LED)
            #define LED_MAX_STRIP_LENGTH 64
        #else
            #define LED_MAX_STRIP_LENGTH 32
        #endif
    #endif

Reply:
@limonspb I guess @blckmn suggestion is the best option as I got the same conclusion. Thanks @ledvinap

Reply:
ok cool, thanks, guys. I have done as suggested.
The only "problem" i see now, sort of off topic:
GOOD: developers/crazy people now can use `LED_MAX_STRIP_LENGTH = 123` as a command line build option locally.
NOT-SO-GOOD: can't use it from the Configurator, because Configurator adds "USE_" to every custom defines.

Maybe it makes sense for the configurator to not add "USE_" for custom defines?

Reply:
Configurator doesn't add it (or shouldn't) but the build processor does. I have added an exclusion list, and any that have an "=" equals in it will not be touched.